Package.swift

```diff
@@ -79,7 +79,7 @@ func firebaseDependency() -> Package.Dependency {
     return .package(url: firebaseURL, branch: "main")
   }
 
-  return .package(url: firebaseURL, from: "11.5.0")
+  return .package(url: firebaseURL, from: "12.0.0")
```

Comment:
If Firebase 12 didn't break anything in Data Connect, should we instead broaden the support range?
```suggestion
  return .package(
      url: firebaseURL,
      "11.5.0" ..< "13.0.0"
    )
```

Reply:
+ @paulb777

Reply:
`from:` should automatically take care of going all the way to < 13.x per this doc. 

https://developer.apple.com/documentation/packagedescription/package/dependency/package(url:from:)#discussion

But yes, we would lose 11.5 - so makes sense the keep the lower bound as - is.

Reply:
Fixed.